cli/src/main/resources/application.yml

```diff
@@ -43,73 +33,106 @@ message:
 spring:
   config:
     activate:
-      on-profile: kafka
+      on-profile: command
 
-hono:
-  kafka:
-    commonClientConfig:
-      bootstrap.servers: localhost:9092
-    consumerConfig:
-      group.id: hono-cli
+command:
+  timeoutInSeconds: 60
 
 ---
+# Configuration for the Hono sandbox - see:
+# https://www.eclipse.org/hono/getting-started/
+# Use this profile together with "receiver" or "command"
 
 spring:
   config:
     activate:
-      on-profile: statistic
+      on-profile: sandbox
 
-statistic:
-  interval: 10000
-  autoreset: false
+hono:
+  client:
+    host: hono.eclipseprojects.io
+    port: 15671
+    tlsEnabled: true
+    username: consumer@HONO
+    password: verysecret
+  kafka:
+    commonClientConfig:
+      bootstrap.servers: hono.eclipseprojects.io:9092
+      security.protocol: SASL_PLAINTEXT
+      sasl.mechanism: SCRAM-SHA-256
+      sasl.jaas.config: "org.apache.kafka.common.security.scram.ScramLoginModule required username=\"hono\" password=\"hono-secret\";"
```

Comment:
isn't it possible to use the same user/pwd for kafka as we do for AMQP?

Reply:
The character '@' does not work in the username. I do not know the exact cause. I am open to other ideas for usernames. 
Apart from that, this needs to be changed in _IoT Packages_ first, I would like to decouple that from this PR.